pkg/controller/pxc/controller.go

```diff
@@ -612,8 +622,12 @@ func (r *ReconcilePerconaXtraDBCluster) deploy(ctx context.Context, cr *api.Perc
 
 	var proxyInits []corev1.Container
 	if cr.CompareVersionWith("1.13.0") >= 0 {
+		initResources := cr.Spec.PXC.Resources
+		if cr.CompareVersionWith("1.14.0") >= 0 {
+			initResources = cr.Spec.InitResources
+		}
```

Comment:
we need to provide a custom build based on 1.13.0. This won't work. We need check if `cr.Spec.InitResources` is empty and use it if not.

Reply:
@egegunes but cr.Spec.InitResources will be always empty. Operator sets it on sts level. I think @pooknull was right.

Reply:
I don't get it. Why will it be always empty? Won't users set it if they want to configure? And if we provide custom build, will the user have `crVersion: 1.14.0`?